packages/block-library/src/separator/transforms.js

```diff
@@ -18,6 +18,17 @@ const transforms = {
 			},
 		},
 	],
+	to: [
+		{
+			type: 'block',
+			blocks: [ 'core/spacer' ], // Transform to Spacer.
+			transform: () => {
+				return createBlock( 'core/spacer', {
+					height: '50px',
```

Comment:
Is the `50px` to achieve a height similar to a separator? The spacer does already have a `100px` default.

It'd be good to include any other mutual attributes. I think `anchor` is the only on that I can see. If a separator has an anchor it should be retained when transforming to the spacer.

Reply:
Thank you for the feedback!

1. The 50px height was an arbitrary choice to mimic a slightly smaller spacing effect than the Spacer block's default 100px. However, I understand that 100px is the default height for a Spacer, so I'm happy to adjust it to the default if that better fits the expected behavior.
2. Great point about the anchor attribute! I will update the transform so that if a Separator block has an anchor, it will be retained when transforming to the Spacer block. This ensures that the transformation keeps user-defined anchors intact.